python/ray/_private/utils.py

```diff
@@ -1900,10 +1897,9 @@ def __exit__(self, exc_type, exc, exc_tb):
         signal.signal = self.orig_signal
         # Restore original SIGINT handler.
         signal.signal(signal.SIGINT, self.orig_sigint_handler)
```

Comment:
I don't feel comfortable keep calling it an **original** signal handler, since now we really mean "the original signal handler, **or the signal handler registered by the user within the context manager**". What about: `self.deferred_signal_handler` ?

Reply:
Updated to `overridden_sigint_handler`, which means the signal handler overridden during the context.

I think `deferred_signal_handler` sounds a little bit weird if no signal handler is registered within the context manager. The original signal handler simply get overridden and then restored. There is no `deferred` registering.

Let me know if you still think `deferred_signal_handler` is better.